packages/@okta/i18n/src/properties/login_fr.properties

```diff
@@ -996,7 +996,7 @@ oie.security.question.questionKey.label = Choisissez une question de s\u00E9curi
 oie.security.question.createQuestion.label = Cr\u00E9er ma propre question de s\u00E9curit\u00E9
 oie.security.question.authenticator.description = Choisissez une question de s\u00E9curit\u00E9 qui sera utilis\u00E9e pour se connecter
 # {0} is a number
-securityQuestion.answer.tooShort.arg = La r\u00E9ponse \u00E0 la question de s\u00E9curit\u00E9 doit contenir au moins {0}\u00A0caract\u00E8res
+securityQuestion.answer.tooShort.arg = La r\u00E9ponse \u00E0 la question de s\u00E9curit\u00E9 doit contenir au moins {0} caract\u00E8res
```

Comment:
Should we revert this no-break-space to actual space change ?

Reply:
similar discussion https://github.com/okta/okta-core/pull/74189#discussion_r1055887315

Reply:
In the english version, we don't have a `no-break-space` though? Does it make sense to update it to be inconsistent with the english file?  

"caract" is just "character" in french so I assume it's just saying "X characters". So I think we are fine : )

Reply:
Lets stay consistent with en